musicbot/bot.py

```diff
@@ -1781,14 +1781,32 @@ def check(message):
 
 
     async def cmd_disconnect(self, server, message):
+        await self.safe_send_message(
+            message.channel,
+            'Disconnecting from %s...' % server,
+            expire_in=30,
+            also_delete=message if self.config.delete_invoking else None
```

Comment:
`expire_in` and `also_delete` are pointless since the bot will not maintain the state of the messages to delete them after restarting.